app/views/journeys/income/AnyOtherIncomeView.scala.html

```diff
@@ -15,29 +15,33 @@
  *@
 
 @import controllers.journeys.income.routes.AnyOtherIncomeController
+@import views.html.templates.helpers.Heading
 
 @this(
     layout: templates.Layout,
     formHelper: FormWithCSRF,
     govukErrorSummary: GovukErrorSummary,
     govukRadios: GovukRadios,
-    govukButton: GovukButton
+    govukButton: GovukButton,
+    heading: Heading
 )
 
-@(form: Form[_], mode: Mode, taxYear: Int)(implicit request: Request[_], messages: Messages)
+@(form: Form[_], mode: Mode, isAgentString: String, taxYear: Int)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(form, messages("anyOtherIncome.title"))) {
+@layout(pageTitle = title(form, messages(s"anyOtherIncome.title.$isAgentString"))) {
 
-    @formHelper(action = AnyOtherIncomeController.onSubmit(taxYear, mode), Symbol("autoComplete") -> "off") {
+    @if(form.errors.nonEmpty) {
+        @govukErrorSummary(ErrorSummaryViewModel(form))
+    }
 
-        @if(form.errors.nonEmpty) {
-            @govukErrorSummary(ErrorSummaryViewModel(form))
-        }
+    @heading(messages(s"anyOtherIncome.title.$isAgentString"), None, extraClasses = "govuk-!-margin-bottom-4")
```

Comment:
Just out-of-interest, what is this `heading` doing? Is the page title not added by line 31?

Reply:
On line 31 changes the page's title (in the browser tab) rather than the page's heading